src/nodedef.h

```diff
@@ -441,6 +441,8 @@ struct ContentFeatures
 	NodeBox node_box;
 	NodeBox selection_box;
 	NodeBox collision_box;
+	// if selection box is bigger then (-BS,-BS,-BS)/2 and (BS,BS,BS)/2 edges
+	bool bigSelectionBox;
```

Comment:
```suggestion
	/// if selection box is bigger then (-BS,-BS,-BS)/2 and (BS,BS,BS)/2 edges
	bool have_big_selection_box;
```
1. https://dev.minetest.net/Code_style_guidelines#Miscellaneous
   (The "local" part of "local variable" could be removed eventually)
2. Doxygen formatting for better IDE experience

Reply:
fixed